executor/executor.go

```diff
@@ -296,6 +297,60 @@ type ShowDDLJobsExec struct {
 	jobs   []*model.Job
 }
 
+// ShowDDLJobQueriesExec represents a show DDL job queries executor.
+// The command query has completed job history sql limit 10.
```

Comment:
The jobs id that is given by 'admin show ddl job queries' statement, only be searched in the latest 10 history jobs.

Reply:
That's what it means.